crates/bevy_winit/src/system.rs

```diff
@@ -298,10 +298,10 @@ pub(crate) fn changed_windows(
             }
 
             if window.ime_position != cache.window.ime_position {
-                winit_window.set_ime_position(LogicalPosition::new(
-                    window.ime_position.x,
-                    window.ime_position.y,
-                ));
+                winit_window.set_ime_cursor_area(
+                    LogicalPosition::new(window.ime_position.x, window.ime_position.y),
+                    PhysicalSize::new(10, 10),
```

Comment:
👀  I think I had a reason about hardcoding `(10,10)`, but that probably should benefit a comment, or a fix.